packages/cpp/src/browser/cpp-commands.ts

```diff
@@ -44,17 +44,17 @@ export const SHOW_CLANGD_REFERENCES: Command = {
 
 export const DUMP_INCLUSIONS: Command = {
     id: 'clangd.dumpinclusions',
-    label: 'C/C++: Dump file inclusions (Debug)'
+    label: 'C/C++: Dump File Inclusions (Debug)'
 };
 
 export const DUMP_INCLUDED_BY: Command = {
     id: 'clangd.dumpincludedby',
-    label: 'C/C++: Dump files including this file (Debug)'
+    label: 'C/C++: Dump Files Including this File (Debug)'
```

Comment:
why this is not capitalize ?

Reply:
I initially had it this way but after verbal review from @lmcbout, we decided that `stop words` such as (a, in, this, an) would not be capitalized. Example: `Reveal in Files`.

Do you think we should instead capitalize?

Reply:
This seems like something we should be consistent about. I remember a few related PR discussions, but I do not see anything on the [coding guidelines](https://github.com/theia-ide/theia/wiki/Coding-Guidelines) page. Personally I would keep ```this``` lowercase.